drivers/gpu/drm/drm_atomic_helper.c

```diff
@@ -2517,6 +2518,14 @@ void drm_atomic_helper_commit_hw_done(struct drm_atomic_state *old_state)
 		complete_all(&old_state->fake_commit->hw_done);
 		complete_all(&old_state->fake_commit->flip_done);
 	}
+
+	if (old_state->restore == 1) {
+		old_state->restore = 0;
+		ktime_t now = ktime_get();
+		u64 period_ns = ktime_to_ns(ktime_sub(now, acpi_resume_begin));
+		drm_info(crtc->dev, "resume completet, resume period %lld,  restore = %d\n",
```

Comment:
the log is confused, should be changed as:
The period from S3 resume to first display atomic commit is: 
Also don't need print the restore value